Gemfile

```diff
@@ -56,7 +56,9 @@ gem "importmap-rails", "~> 2.1"
 ##### VIEWS/CONTROLLERS #####
 
 # Adds filter support to models and views.
-gem "filterrific"
+# TODO: Stop pinning filterrific to git once there is a release with the following commit:
+# https://github.com/jhund/filterrific/commit/bb1ab756575674a19e1632c9eeb5eda859a8cb8a
+gem "filterrific", git: "https://github.com/jhund/filterrific", branch: "master"
```

Comment:
@coalest looks like this has been released!

Reply:
Great! I switched it back and upgraded to the new version.